CRR/ReplicationStatusUpdater.js

```diff
@@ -352,7 +352,7 @@ class ReplicationStatusUpdater {
                     });
                 },
             ),
-            () => {
+            callback => {
```

Comment:
best to just make the function `async` :


```suggestion
            async () => {
```